README.md

```diff
@@ -1,2 +1,97 @@
-# Hello-Microverse
-In this project, I will set up a "Hello world" repository. No complex coding is required for this exercise. My goal here is to master all of the tools and best practices I have learned about in previous steps. I will be using them in all Microverse projects and most likely in my future job.
+# HTML & CSS3 Course
+
+If you are not familiar with linters, read [root level README](../README.md).
+
+## Set-up GitHub Actions
+
```

Comment:
# Hello @zhadier,

I made an embarrassing mistake yesterday by merging this pull request. 

<img src="https://giphy.com/clips/thefastsaga-fast-and-furious-saga-fate-of-the-HKu3kNqtBWGynqAcfj"/>

## As I can see I've got three different paths to follow:

1. Create a new pull request
2. Make a new repository
3. Push directly to the main branch

Since you specify not to open a new pull request, I will try 2 and 3 options.

Please tell me if I have to do something else.

Reply:
# Hi @zhadier,

I messed up (even more) my repository. This time Linters didn't work. 
I've requested a new review of this other [pull request](https://github.com/MarcoOquendoC/HelloMicroverse/pull/1) in a new [repository](https://github.com/MarcoOquendoC/HelloMicroverse.git).

Thank you for your time.

Marco Oquendo.